examples/runtime-example/src/runtime.ts

```diff
@@ -20,9 +20,10 @@ export const createRuntime = async (client: Knex) => {
   const backend = new GraphQLBackendCreator(schemaText, jsonConfig.graphqlCRUD);
   const dbClientProvider = new PgKnexDBDataProvider(client);
 
-  const dbInitialization = new UpdateDatabaseIfChanges(client, jsonConfig.folders.migrations);
-
-  await migrate(schemaText, dbInitialization);
+  const ops = await migrate({
```

Comment:
API is similar

Reply:
What is schema text here ? Model? Are we stripping out migration directives

Reply:
Input model. Why would we strip out migration directives when we need them to run the migration?